assets/airports/kmia.json

```diff
@@ -42,495 +42,1258 @@
+            ]
         },
         "FLIPR4": {
             "icao": "FLIPR4",
             "name": "Flipper Four",
             "entryPoints": {
-                "ZFP": ["ZFP", "LUCSS", "FOWEE"],
-                "ZQA": ["ZQA", "NICKO", "TINKY", "LUCSS", "FOWEE"],
-                "HODGY": ["HODGY", "JAYEE", "PEKRE", "TINKY", "LUCSS", "FOWEE"],
-                "URSUS": [["URSUS", "S280"], "ZOLLA", "FOWEE"]
+                "HODGY": ["HODGY", "JAYEE", "PEKRE", "TINKY", "LUCSS"],
+                "URSUS": ["URSUS", "ZOLLA"],
+                "ZFP": ["ZFP", "LUCSS"],
+                "ZQA": ["ZQA", "NICKO", "TINKY", "LUCSS"]
             },
-            "body": [["FLIPR", "A120|S280"], "LUVLY", "LECIT", "JODPO"],
-            "draw": [["ZFP*","LUCSS","FOWEE"],["ZQA*","NICKO","TINKY","LUCSS","FOWEE"],["HODGY*","JAYEE","PEKRE","TINKY","LUCSS","FOWEE"],["URSUS*","ZOLLA","FOWEE"],["FOWEE","FLIPR","LUVLY","LECIT","JODPO"]]
+            "body": ["FOWEE", "FLIPR", "LUVLY", "LECIT", "JODPO"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
+            },
+            "draw": [
+                ["HODGY*", "JAYEE", "PEKRE", "TINKY", "LUCSS", "FOWEE", "FLIPR", "LUVLY", "LECIT", "JODPO"],
+                ["URSUS*", "ZOLLA", "FOWEE"],
+                ["ZFP*", "LUCSS", "FOWEE"],
+                ["ZQA*", "NICKO", "TINKY", "LUCSS"]
+            ]
         },
         "FOWEE8": {
             "icao": "FOWEE8",
             "name": "Fowey Eight",
             "entryPoints": {
-                "ZFP": ["ZFP", "LUCSS", "FOWEE"],
-                "ZQA": ["ZQA", "TINKY", "LUCSS", "FOWEE"],
-                "HODGY": ["HODGY", "TINKY", "LUCSS", "FOWEE"],
-                "URSUS": ["URSUS", "FOWEE"]
+                "HODGY": ["HODGY", "TINKY", "LUCSS"],
+                "URSUS": ["URSUS"],
+                "ZFP": ["ZFP", "LUCSS"],
+                "ZQA": ["ZQA", "TINKY", "LUCSS"]
             },
-            "body": [["JUNUR", "A120|S280"], "LUVLY", "DHP"],
-            "draw": [["ZFP*","LUCSS","FOWEE"],["ZQA*","TINKY","LUCSS","FOWEE"],["HODGY*","TINKY","LUCSS","FOWEE"],["URSUS*","FOWEE"],["FOWEE","JUNUR","LUVLY","DHP"]]
+            "body": ["FOWEE", "JUNUR", "LUVLY", "DHP"],
+            "rwy": {
+                "KMIA8L": [],
+                "KMIA8R": [],
+                "KMIA9": [],
+                "KMIA12": [],
+                "KMIA26L": [],
+                "KMIA26R": [],
+                "KMIA27": [],
+                "KMIA30": []
+            },
+            "draw": [
+                ["HODGY*", "TINKY", "LUCSS", "FOWEE", "JUNUR", "LUVLY", "DHP"],
+                ["URSUS*", "FOWEE"],
+                ["ZFP*", "LUCSS", "FOWEE"],
+                ["ZQA*", "TINKY", "LUCSS"]
+            ]
         },
         "HILEY6": {
```

Comment:
HILEY6 is now HILEY7

Reply:
reverified and updated all SIDs/STARs.

done